sources/oracle/toddl.go

```diff
@@ -25,8 +25,8 @@ import (
 )
 
 var (
-	timestampReg = regexp.MustCompile(`TIMESTAMP`)
-	intervalReg  = regexp.MustCompile(`INTERVAL`)
+	TimestampReg = regexp.MustCompile(`TIMESTAMP`)
```

Comment:
does this need to be public ?

Reply:
It does not need to be. Updated the names.